README.md

```diff
@@ -2,62 +2,111 @@
 
 # End to end tests for Archive.org using [Playwright](https://playwright.dev/)
 
+
 ## (Optional) BrowserStack Local Setup
 
-- login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
+- Login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
 
 
-## Running tests locally
+## Local Setup
 
-- install dependencies:
+- Install dependencies:
 
     `npm i`
 
-- install Playwright browser libs:
+- Install Playwright browser libs:
 
     `npx playwright install`
 
-- run all the tests in headless mode and generate 1 whole test report:
+- Create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
+
+    - this is required to run tests with loggedIn flows
+
+
+## Running tests
+
+- Run all tests: `npm run test`
+
+- Run tests by category:
+
+    - This will run a custom script that can take different shell arguments/parameters for the following:
+        > Note:
+        >   - The arrangement/order for passing of arguments can be jumbled for flexibility
+        >   - The script is somewhat patterned with the current Playwright commands that uses `--` convention
+        >   - It's important to add `--` convention in CLI to separate the arguments from the `npm script` to be passed to the custom script
```

Comment:
QUESTION: The use of this double-hyphen is ambiguous in the examples below. Sometimes it comes before the test type, e.g. "about", and sometimes after. Also, all meta arguments, e.g. "--headed", are already distinguished by the double-hyphens that are prepended to the option name. So what is the need for these independent "--"?

Reply:
The initial implementation with args starts with `--` like: `--headed` and `--debug` was just to append the existing Playwright arguments with the custom command.

Reply:
updated the code for consistency